wireguard-install.sh

```diff
@@ -62,6 +80,69 @@ AllowedIPs = $CLIENT_WG_IPV4/32,$CLIENT_WG_IPV6/128" >>"/etc/wireguard/$SERVER_W
 	echo "It is also available in $HOME/$SERVER_WG_NIC-client-$CLIENT_NAME.conf"
 }
 
+function uninstall() {
+
+## To uninstall wireguard and its setup ##
+
+
+#Check OS type
+checkOS
+
+#get WireGuard interface name
+SERVER_WG_NIC="wg0"
+read -rp "WireGuard interface name: " -e -i "$SERVER_WG_NIC" SERVER_WG_NIC
+
+ip link set down "$SERVER_WG_NIC"		#stop wireguard interface
+
+#Remove wireguard tool and qrencode
+if [[ $OS == 'ubuntu' ]]; then
+    apt-get remove --purge -y wireguard qrencode
```

Comment:
I'm not sure about uninstalling qrencode, it might be previously installed by the user.

Reply:
In that case, We can include a prompt to ask from user if they want to remove qrencode with default value as y.

Reply:
Wait for @angristan on this